client/src/components/DataFiles/DataFilesModals/DataFilesModalTables/DataFilesModalListingTable.js

```diff
@@ -170,7 +170,7 @@ const DataFilesModalListingTable = ({
       const currentFolderEntry = {
         name: isNotRoot
           ? getCurrentDirectory(params.path)
-          : findSystemDisplayName(systemList, params.system),
+          : findSystemDisplayName(systemList, params.system, !isNotRoot),
```

Comment:
i'm not following this change here and the related change in findSystemDisplayName.

like wouldn't here, the `!isNotRoot` always be `true`.   so why add more logic in findSystemDIsplayName.    

Not certain if this comment makes sense.  Getting tired 😴

Reply:
He's passing a true to that function.